conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+bot:
+  inspection: update-grayskull
```

Comment:
Run Grayskull on version updates and include any dependency changes it finds. Hopefully helps smooth out version updates on the dependency side